app/components/visualizer/main-viz/main-viz.jsx

```diff
@@ -0,0 +1,3 @@
+import Chart from "./templates/Chart.jsx"
```

Comment:
Is there a way to make this a bit more dynamic? I'm not crazy about how "hard coded" it feels at the moment - I understand it is a WIP